boilerplate_app/serializers.py

```diff
@@ -63,4 +65,16 @@ class ProjectsListSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Projects
-        fields = ('id', 'project_name', 'user')
\ No newline at end of file
+        fields = ('id', 'project_name', 'user')
+
+#Time entry serializer for processing json files
+class TimeEntryCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TimeEntry
+        fields = ('account',)
+
+    def create(self, validated_data):
```

Comment:
Oh I see. So you are passing the user Id in here so that the db knows that. but a security problem is that, doesn't this mean anyone can creat a `timeEntry` for anyone else? How would you fix that?